Xplat/src/main/resources/assets/botania/lang/en_us.json

```diff
@@ -3132,7 +3132,7 @@
 
   "botania.entry.knockbackBelt": "Tectonic Girdle",
   "botania.tagline.knockbackBelt": "Prevent all knockback",
-  "botania.page.knockbackBelt0": "By harnessing the (relative) stability of tectonic forces, the $(item)Tectonic Girdle$(0) negates any $(thing)Knockback$(0) applied to its wearer from outside attacks.",
+  "botania.page.knockbackBelt0": "By harnessing the (relative) stability of tectonic forces, the $(item)Tectonic Girdle$(0) negates any $(thing)Knockback$(0) applied to its wearer from outside attacks and all recoil from the $(item)Mana Blaster$(0).",
```

Comment:
link to the entry?

Reply:
you'll need to be more specific on what you mean by this :p

Reply:
lexicon entries can link to other ones, and we try to make it so that items that reference other items have a clickable link to that entry. I was asking if you could add a link to the Mana Blaster entry.
Should be something like $(l:botania:mana_blaster)Mana Blaster$(/l), or whatever the entry ID is.

Reply:
ah, I see, yes I'll do that